### DISCUSSION TITLE
Environment creation for off-road autonomy

### DISCUSSION THREAD

**Discussion by mayureshb7** [carla-simulator/carla#5448] [General]
Hi Folks,

1) Does CARLA allow the creation of a custom 3D environment or map for off-road simulation, and if yes, how?
2) I believe the addition of Carsim to CARLA has increased vehicle dynamics parameters and data acquisition capabilities. Can I include the same for off-road autonomy if an environment can be created?

Regards,
Mayuresh

**Comment by ErvinGrimaldi03** [comment]
Did you ever figure out?

**Reply by mayureshb7** [reply]
Found this never used https://github.com/lssac7778/Off-road-Benchmark
